src/fft_shift.cu

```diff
@@ -0,0 +1,80 @@
+#include <bifrost/fft_shift.h>
+#include <cuComplex.h>
+#include "assert.hpp"
+#include "trace.hpp"
+#include "utils.hpp"
+#include "cuda.hpp"
+#include "cuda/stream.hpp"
+
+// This explicily fft shifts a 2D space. 
+
+template <typename T>
+__global__ void fft_shift_kernel_2d_k(T *grid, int size){
+
+  int x = blockIdx.x * blockDim.x + threadIdx.x;
+  int y = blockIdx.y * blockDim.y + threadIdx.y;
+
+  if(x<size/2 && y <size){
+
+    int ix0 = y * size + x;
+    int ix1 = (ix0 + (size + 1) * (size/2)) % (size*size);
+
+    T temp = grid[ix0];
+    grid[ix0] = grid[ix1];
+    grid[ix1] = temp;
```

Comment:
Does this only shift in one dimension? It looks like it only shifts x, and I don't see where y gets shifted.

Reply:
No idea, I did not write this. May be James has an answer. We had to manually copy stuff from Jame's repo as per the earlier install instructions. I guess it just stayed

Reply:
`fft_shift.cu` doesn't seem to be in ledatelescope bifrost.  Maybe this isn't needed and should be removed?